### PR TITLE
Roboclaw module docs: make command code bloc

### PR DESCRIPTION
**Pull request by hamishwillee**
This gets generated to docs, which render better if the text is a code block

### CODE REVIEW

File: src/drivers/roboclaw/roboclaw_main.cpp [hunk 1]
```diff
@@ -108,8 +108,9 @@ the driver terminates immediately.
 ### Examples
 
 The command to start this driver is:
-
+```
  $ roboclaw start <device> <baud>
```
Comment (bkueng) [inline src/drivers/roboclaw/roboclaw_main.cpp:112]: If you remove the space in front of ` $ roboclaw start <device> <baud>` it should also convert to a block.

**Comment by mrpollo** [comment]
@bkueng hey is your suggestion the way you want to go about this PR? if so I can quickly add a commit to resolve, otherwise I'm ready to merge this

**Comment by bkueng** [comment]
> @bkueng hey is your suggestion the way you want to go about this PR?

Yes it's the preferred way of doing it.

**Comment by MaEtUgR** [comment]
Closing since it was removed in https://github.com/PX4/PX4-Autopilot/pull/21239
We can add the driver back once it's used again. Might happen sooner than later.

**Pull request closed without merging**